README.md

```diff
@@ -444,6 +444,20 @@ App behavior parameters
 
   - Default: *[Number]* `30000` (30 seconds).
 
+- `routers`: An array of objects that define express routers.
+  - `prefix`: *[String]* The URL path prefix for the router to use.
+  - `controllers`: *[Array]* List of files or directories relative to the controllers directory to use with the defined prefix.
+  - Default: *[Boolean]* `false`.
+  - Example:
+
+    ```json
+    [{
+      "prefix": "/route",
+      "controllers": ["controller.js", "directory"]
+    }]
+    ```
+  **NOTE** When using `routers` roosevelt will modify the `app` argument within controller files to use an express `router`. The app object will be the second argument inside of those controllers.
```

Comment:
This line can go since we're gonna universalize the arguments to always be router and app

Reply:
This is good now